L/libseat/bundled/patches/fix_crosscompile.patch

```diff
@@ -0,0 +1,33 @@
+--- meson.build
```

Comment:
It'd be good to upstream as much as possible of this patch

Reply:
Agree, I am working on submitting a patch

Reply:
Hope I did that correctly: https://lists.sr.ht/~kennylevinsen/seatd-devel/patches/23402

Reply:
Why did you have to comment some flags?  Was something failing with them?

Reply:
Yes, I think they were only added in a later gcc version, so they errored in gcc 4

Reply:
Well, we can use a later gcc if that's the case

Reply:
I will look into whether that can be made conditional upstream as well later today

Reply:
> Well, we can use a later gcc if that's the case

I just thought we preferred older gcc versions if at all possible

Reply:
Yes, but between 4 and 5 there isn't a huge difference.  The problem is when people insist using GCC 10

Reply:
I think we'd need 7 at least.

Reply:
Also, this seems to be a C package, for C we have fewer compatibility concerns.  It's C++ which is more problematic

Reply:
Is it worth changing, so we carry around fewer patches?

Reply:
Yeah, I think so.  As I said, for C we have fewer compatibility concerns anyways, and this is kind of an implicit requirement of the upstream project, even though we could easily work around it.